content/automate/ManageIQ/System/CommonMethods/StateMachineMethods.class/transformation_plan_vm_finished.yaml

```diff
@@ -0,0 +1,13 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: transformation_plan_vm_finished
+    inherits: 
+    description: 
+  fields:
+  - execute:
+      value: task_finished(object => 'transformation_plan_task', message => 'VM Migrated
```

Comment:
@bzwei Does this ```object => 'transformation_plan_task'``` make sense here?

Reply:
`service_template_transformation_plan_task`